core/types/wo.go

```diff
@@ -358,8 +361,8 @@ func (wo *WorkObject) TransactionsWithReceipts() []*Transaction {
 // This also returns all transactions that have an associated receipt.
 func (wo *WorkObject) QuaiTransactionsWithoutCoinbase() []*Transaction {
```

Comment:
This function is unused and wrong, I think it should be deleted

Reply:
deleted